nuxeo-core/nuxeo-core-test/src/test/resources/OSGI-INF/csvexport-config-override.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.bulk.config.test" version="1.0.0">
+
+  <require>org.nuxeo.ecm.core.bulk.config</require>
+
+  <extension target="org.nuxeo.ecm.core.bulk" point="actions">
+    <action name="csvExport" bucketSize="1" batchSize="1" />
```

Comment:
instead of using a contribution you could simply change the bulk command the builder enable to set the batch and bucket size

Reply:
The Bulk scroller computation was still relying on the parameters from the descriptor instead of the one from the command.